### ISSUE TITLE
Starts on boot of Windows 10, even when checkbox is unselected

### ISSUE THREAD

**Issue by janacm** [stefansundin/altdrag#84]
Altdrag starts in a state which does not work when I logon to Windows 10. It works fine when I end-task and then re-open. 

![image](https://user-images.githubusercontent.com/2055312/32962365-2f9acb7a-cb9a-11e7-93e2-3824032869b4.png)

**Comment by stefansundin** [comment]
That is strange. Are you sure you haven't created a task scheduler to launch AltDrag? I think that's the only case that I've seen the program launch in an unresponsive state.

**Comment by janacm** [comment]
Yes, this is a newly imaged Windows 10 box, and altdrag is one of the first
things I installed (cuz its a great app!). So no task scheduler is bringing
it up. Is there any logs I can provide you?

**Comment by janacm** [comment]
If this issue ever gets solved, then I hope to come back to using this application. In the meantime, if anyone is looking for an alternative, this has been working out for me: http://corz.org/windows/software/accessories/KDE-resizing-moving-for-Windows.php

**Comment by RamonUnch** [comment]
@janacm Hopefully this issue should be fixed on my fork from AltDrag
https://github.com/RamonUnch/AltDrag/releases/
give it a try if you still use Windows and preferred AltDrag to the ahk script
